content/03-reference/01-tools-and-interfaces/02-prisma-client/030-crud.mdx

```diff
@@ -779,7 +807,13 @@ export type FirstNameLastNameCompoundUniqueInput = {
 
 ### Return type
 
-`delete` returns a plain old JavaScript object. 
+`delete` returns a plain old JavaScript object or throws a `RecordNotFound` exception if the unique identifier does not exist:
+
+```terminal
+Error for binding '0': RecordNotFound("Record to delete does not exist.")
```

Comment:
Offtopic:
@eemmiillyy, this is an example for a non optimal error message as it includes information that is not relevant for the user `binding '0'` - might be worth creating an issue for a _test_ to test this, and one for improving the error message some time in the future to something more user friendly. I would also be curious, how this compares between databases (MySQL, Postgres, SQLite).